node/src/Pos/Client/Txp/Util.hs

```diff
@@ -141,6 +141,20 @@ isCheckedTxError = \case
 -- Tx creation
 -----------------------------------------------------------------------------
 
+-- | Specifies the way Uxtos are going to be grouped.
+data InputSelectionPolicy
+    = OptimizeForSecurity  -- ^ Spend everything from the address
+    | OptimizeForSize      -- ^ No grouping
+    deriving (Show, Eq, Generic)
+
+instance Buildable InputSelectionPolicy where
+    build = \case
+        OptimizeForSecurity -> "securely"
+        OptimizeForSize -> "simple"
+
+instance Default InputSelectionPolicy where
+    def = OptimizeForSize
```

Comment:
Yes, that's the crucial bit, we are effectively defaulting to `OptimiseForSize`, so that passing `null` to the payment endpoint will ensure we will call the `plainInputPicker`. Well done, @akegalj .